### ISSUE TITLE
SMTP_SSL is missing from .env.example

### ISSUE THREAD

**Issue by hackhowtofaq** [huginn/huginn#2796]
I was getting the following error on send mail agent

```
net/protocol.rb:225:in `rbuf_fill'
```

Then I noticed that in **action_mailer.rb** an **ENV["SMTP_SSL"]** is expected

https://github.com/huginn/huginn/blob/13d0ca344ae72515795645db058e7c329fd3de5d/config/initializers/action_mailer.rb#L7

but I cannot find where is this defined in **.env.example** or elsewhere.

I couldn't send an email with my yahoo account, so I added in my **.env**

```
SMTP_SSL=true
```
and email worked!

**Comment by dsander** [comment]
Good catch, do you mind sending a PR that adds it to the `.env.example` file?

**Comment by hackhowtofaq** [comment]
Yeap no problem
